customer.rb

```diff
@@ -0,0 +1,149 @@
+require_relative '../Bank/validetion.rb'
+require_relative '../Bank/account.rb'
+require_relative '../Bank/transaction.rb'
+
+
+class Customer 
+
+  @@status = 0 
```

Comment:
Don't use a class variable for session storage. This code means all customers are login when it's 1 and all are logged out if set to 0.